include/oxen/quic/connection.hpp

```diff
@@ -405,7 +416,13 @@ namespace oxen::quic
         std::atomic<bool> _close_quietly{false};
         std::atomic<bool> _is_validated{false};
 
-        ustring remote_pubkey;
+        ustring remote_pubkey{};
+
+        std::set<int64_t> _early_streams;
```

Comment:
AFAICT this is never actually used for lookup, but rather just as a simple container that gets iterated through.  I think a vector would suffice for that instead and be marginally simpler?

Reply:
Yep, never used for it. The container ensures that, and the monotonic increasing stream ID allows us to emplace_hint at the back of the container. I can make it a vector if you feel it should be one!